content/en/docs/tasks/administer-cluster/dns-horizontal-autoscaling.md

```diff
@@ -12,6 +12,8 @@ Kubernetes cluster.
 
 * {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+* This guide assumes your nodes use the AMD64 or Intel 64 CPU architecture
```

Comment:
curious: why you feel we need to specify it ?

Reply:
If you're using a less common instruction set on your nodes, `cluster-proportional-autoscaler-amd64` won't execute.

Reply:
(portability of compiled code package between CPU architectures seems like a thing that's harder to do with OCI containerised apps than on, say, Fedora or macOS)

Reply:
okay, thx for detailed info.